lib/evm_database.rb

```diff
@@ -16,6 +16,12 @@ class EvmDatabase
     MiqReport
     RssFeed
     MiqWidget
+    MiqAction
+    MiqEventDefinition
+    Condition
+    MiqPolicy
+    MiqPolicyContent
+    MiqPolicySet
```

Comment:
Oh I see...hmmm...yeah an ORDERED_CLASSES seems ok.  Another thing that we do elsewhere ,when ordering is important, is to put a seed call to the dependent methods right in the .seed methods itself.  In other words, if `Foo.seed` depends on `Bar.seed` we would write it as follows:

```ruby
class Foo
  def self.seed
    Bar.seed
    # Seeding code for Foo
  end
end

class Bar
  def self.seed
    # Seeding code for Bar
  end
end
```

Since seeding should be reentrant, this should work fine (although it may not be the most effiecient thing depending on how the various seeding code is written.